rules/ensure-external-secrets-storage-is-in-use/rule.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ensure-external-secrets-storage-is-in-use",
+  "attributes": {
+    "armoBuiltin": true
+  },
+  "ruleLanguage": "Rego",
+  "match": [
+    {
+      "apiGroups": [
+        ""
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+        "Pod",
+        "CronJob",
+        "Deployment",
+        "ReplicaSet",
+        "DaemonSet",
+        "StatefulSet",
+        "Job"
```

Comment:
some resources have different apigroups, see similar rules that use these resources

Reply:
Fixed.